README.md

```diff
@@ -389,6 +389,34 @@ So the comparisons in Ruby would be:
 
 The fields do match successfully, but note the the first comparison matches `id` on `id` and `course_id` so the match would be wider than expected in some instances.
 
+## Page Caching
+
```

Comment:
@asatwal is it worth adding something like 'Some applications use rack middleware to cache pages. If your application does not use this you can disregard this section.' here?